### ISSUE TITLE
Notification edges

### ISSUE THREAD

**Issue by Dil3mm4** [wasdennnoch/AndroidN-ify#1285]
**tl;dr I know that it's no longer developed, but if someone has some spare-time would be nice to have a fix.**
**Description**
Notification edges.
Dunno if it's intended to be like that, I was just wondering if it was a bug or not.
In the stable version the graphics was "continous", in this one, there's a small border. 
**Module version (build version)**
801 build
**OS**
5.0.2 - LG Stock
**xPosed version**
84
**Infos**
If there's something else needed just ask.
**Screenshot**
![screenshot 09 2fgen 2f2017 17-31-11](https://cloud.githubusercontent.com/assets/21145257/21775240/9757d082-d695-11e6-8b57-f97d3d6c2da8.png) 
@paphonbth @Maxr1998

**Comment by sargijeh** [comment]
It is the nature of most lollipop roms, they can't fix it I think because I have this issue from the first build on 5.1.1

**Comment by dil3mm4** [comment]
Nope. on the stable seems to work correctly, at least for me .

**Comment by wasdennnoch** [comment]
#935
